rev_news/drafts/edition-67.md

```diff
@@ -43,6 +43,10 @@ __Various__
 
 __Light reading__
 
+Arista Networks, a Fortune 500 company, has rolled out mandatory commit signing across their git repositories. Read about their journey at https://eos.arista.com/commit-signing-with-git-at-enterprise-scale/ and learn about how they:
```

Comment:
I believe this should be turned into a Markdown link -- `[text](link)`. Otherwise, a change similar to the one done in 4a4c1943d2dbbb85a22261e03a285739407504d8 might be needed.